rules/concepts.md

```diff
@@ -48,6 +48,151 @@ In some cases, it is useful to have a rule that is only manually triggered.
 Sometimes there is advantage in a rule that calls another rule.
 In both cases, the rule has no triggers and no conditions.
 
+## Triggers
 
+Now that we know what rules are, we need to have something to make them run when something happens.
+This is when we need triggers.
 
+There are different categories of rule triggers:
 
+| Trigger type      | Description                                                            |
+|-------------------|------------------------------------------------------------------------|
+| **Item**-based    | Those react on commands or state updates for the Item                  |
+| **Group**-based   | Those react on commands or state updates for member Items of the Group |
+| **Time**-based    | Those react at special times, e.g. at midnight, every hour, etc.       |
+| **Channel**-based | Those react on trigger channels provided by some add-ons               |
+| **Thing**-based   | Those react on thing status, e.g. change from `ONLINE` to `OFFLINE`    |
+| **System**-based  | Those react on certain system statuses, e.g. startup completed         |
+
+::: tip Triggers and Events - What's the difference?
+You might have heard or will hear about events and triggers, and wonder about the difference.
+
+An event is something that happens and is detectable by openHAB.
+The most important events for rules are time events (e.g. midnight), system events (e.g. startup completed), Item state events and Item command events.
+There are some more events, but it's not necessary to mention them here.
+
+In contrast to this, a trigger identifies one of those events and makes the rule run.
+:::
+
+Here are the details for each trigger type:
+
+### **Item**-based Triggers
+
+You can listen to commands for a specific item, on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific command/status or any.
+
+A simplistic explanation of the differences between `command` and `update` can be found in the article about [openHAB core actions](/docs/configuration/actions.html#event-bus-actions).
```

Comment:
> But I think we will be better off if we move and augment the description of the difference between updates and commands to Items and reference that here, or put it here and reference to here in the rewritten Actions and Rules pages (perhaps even the Items page too).

I would prefer to have this at Items, as it is the most logical page for me.

> Does the ":::tip" show up in the table of contents and support direct linking? If so, maybe it could be another tip section.

I don‘t know, sorry. Shouldn‘t matter when we have the explanation at Items.

I will create an issue to add that explanation to Items.

Reply:
I just checked the docs, https://www.openhab.org/docs/configuration/items.html#command-vs-status should work for us.
Maybe this (as well as the whole Items page) needs some updates (seems like it was written when openHAB 3 was not out) and rewording (status vs. state).